test/test_lazy.py

```diff
@@ -0,0 +1,90 @@
+import torch
+from common_utils import TestCase, run_tests, skipIfRocm, do_test_dtypes, do_test_empty_full, load_tests
+import time
+
+class TestLazy(TestCase):
+  def test_bench(self):
```

Comment:
I'm surprised lint isn't failing you here, because we're supposed to use four spaces for Python indenting right?